plugins/emmc/fu-emmc-device.c

```diff
@@ -372,13 +386,18 @@ fu_emmc_device_write_firmware(FuDevice *device,
 		return FALSE;
 	fw_size = g_bytes_get_size(fw);
 
+	sector_size = self->write_block_size ? : self->sect_size;
+
+	/*  mode operation codes are supported */
+	check_sect_done = ext_csd[EXT_CSD_FFU_FEATURES];
```

Comment:
do you mean something like `ext_csd[EXT_CSD_FFU_FEATURES] > 0` -- i'm unsure what eh gboolean would mean if it's set to something like 253 :)

Reply:
Thanks.  Will update.
Also looking into it further, only bit 0 in this register is valid, all other 7 bits are reserved, so further masking is needed as well.